packages/angular_devkit/schematics/tasks/index.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import { TaskExecutorFactory } from '@angular-devkit/schematics';
+import { NodePackageTaskFactoryOptions } from './node-package/options';
+
+export { NodePackageInstallTask } from './node-package/install-task';
+export { NodePackageLinkTask } from './node-package/link-task';
+
+export class BuiltinTask {
+  static readonly NodePackage: TaskExecutorFactory<NodePackageTaskFactoryOptions> = {
+    name: 'node-package',
```

Comment:
Could you export a list of names as well, so people can refer to the name as a token? Basically, if we rename `node-package` to something else we need tools to rename it as well, but if we export it they can just use the exported constant. Could be namespaced as well.

Reply:
`BuiltinTaskExecutor.NodePackage.name` will provide the name.  I also centralized the name inside `./node-packge/options.ts` so the string itself is only referenced once in the code.